README.md

```diff
@@ -16,13 +16,25 @@ Pulsar simplifies the farming process on Subspace Network.
 
 Instead of running a terminal instance for the farmer, and running another terminal instance for the node, now you can run a SINGLE terminal instance to farm!
 
+## Build
```

Comment:
```suggestion
## Build from Source
```

It might be helpful to add a note that these instructions are optional and primarily aimed at developers who want to build from source but that most users will use pre-built binaries from [releases](https://github.com/subspace/pulsar/releases). Also, I think this should be at the end of the document since it is the non-standard process.